Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM python:3.8-slim-buster
+FROM python:3.9-slim
 WORKDIR /usr/src/app
 COPY requirements_old.txt ./
 RUN pip3 install --upgrade pip
```

Comment:
I think it's redundant and can be removed

Reply:
Ok